ci/tests/026-cargo-audit.sh

```diff
@@ -2,19 +2,12 @@
 
 set -uo pipefail
 echo -e "Start cargo audit\n"
-cargo audit --json > /tmp/audit.json
-jq '.' /tmp/audit.json
-
-
-VULNERABILITIES="$(jq '.vulnerabilities.found' /tmp/audit.json)"
-
 
 if [ "$CARGO_AUDIT_EXIT_ON_ERROR" = "false" ]; then
-  echo -e "\nCargo audit disabled"
-elif [ "$VULNERABILITIES" = "true" ]; then
-  echo -e "\nVulnerabilities have been found"
-  jq '.vulnerabilities' /tmp/audit.json
-  exit 1
-else
-  echo -e "\nNo vulnerabilities have been found"
+  echo -e "\nCargo audit error disabled"
 fi
+
+# Ok: if we want to ignore the `cargo audit` fail but we would see the report
+# we use $CARGO_AUDIT_EXIT_ON_ERROR set to false. In this case we ignore the
+# `cargo audit` output value and we return always `true`.
+cargo audit || ! `$CARGO_AUDIT_EXIT_ON_ERROR`
```

Comment:
Nit, non blocking.
```suggestion
# shellcheck disable=SC2092
cargo audit || ! "$($CARGO_AUDIT_EXIT_ON_ERROR)"
```
All shell scripts should be linted and warnings addressed. To lint use `./ci/devtools/lint_ci.sh`.

Reply:
Ok... thx for the tip!!! I've fixed the lint's advisories and add a if branch to prevent arbitrary code execution... admit only true|false values for `CARGO_AUDIT_EXIT_ON_ERROR` variable.